server/bin/gold/test-18.txt

```diff
@@ -0,0 +1,223 @@
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/BACKED-UP/tarball_config-00_1970.01.01T00.00.40.tar.xz
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/TO-RE-UNPACK/tarball_config-00_1970.01.01T00.00.40.tar.xz
+
+Restored 3 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz
+
+Restored 3 tar balls from backup volume, /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup (ended at: 42.0, elapsed (0.00)
+--- Finished pbench-restore-and-unpack-tarballs (status=0)
++++ Running pbench-unpack-tarballs none re-unpack
+Template:  pbench-unittests.v3.server-reports
+Index:  pbench-unittests.v3.server-reports.1970-01 1
+len(actions) = 1
+[
+    {
+        "_id": "5ca1ab1e70015f100dedfab1ed0ff1ce",
+        "_index": "pbench-unittests.v3.server-reports.1970-01",
+        "_op_type": "create",
+        "_source": {
+            "@generated-by": {
+                "commit_id": "unit-test",
+                "group_id": 43,
+                "hostname": "example.com",
+                "pid": 42,
+                "user_id": 44,
+                "version": ""
+            },
+            "@timestamp": "1970-01-01T00:00:42",
+            "chunk_id": 1,
+            "doctype": "status",
+            "name": "pbench-re-unpack-tarballs",
+            "text": "pbench-re-unpack-tarballs.run-1970-01-01T00:00:42-UTC(unit-test) - w/ 0 errors\nProcessed 0 result tar balls, 0 successfully, 0 warnings, 0 errors, and 0 duplicates\n\n",
+            "total_chunks": 1,
+            "total_size": 165
+        },
+        "_type": "pbench-server-reports"
+    }
+]
+--- Finished pbench-unpack-tarballs (status=0)
++++ Running unit test audit
+Template:  pbench-unittests.v3.server-reports
+Index:  pbench-unittests.v3.server-reports.1970-01 1
+len(actions) = 1
+[
+    {
+        "_id": "5ca1ab1e70015f100dedfab1ed0ff1ce",
+        "_index": "pbench-unittests.v3.server-reports.1970-01",
+        "_op_type": "create",
+        "_source": {
+            "@generated-by": {
+                "commit_id": "unit-test",
+                "group_id": 43,
+                "hostname": "example.com",
+                "pid": 42,
+                "user_id": 44,
+                "version": ""
+            },
+            "@timestamp": "1970-01-01T00:00:42",
+            "chunk_id": 1,
+            "doctype": "status",
+            "name": "pbench-audit-server",
+            "text": "pbench-audit-server.run-1970-01-01T00:00:42-UTC(unit-test)\n",
+            "total_chunks": 1,
+            "total_size": 59
+        },
+        "_type": "pbench-server-reports"
+    }
+]
+--- Finished unit test audit (status=0)
```

Comment:
What produces this report...and, more to the point, why is it being produced by a dry run?

Reply:
This is the standard "audit server" report run by the harness after each test.